R/utils_get_db.R

```diff
@@ -152,7 +152,7 @@ get_mm_data <- function(pkg_id, db_name = golem::get_golem_options('assessment_d
     "SELECT metric.name, metric.long_name, metric.description, metric.is_perc,
                     metric.is_url, package_metrics.value
                     FROM metric
-                    INNER JOIN package_metrics ON metric.id = package_metrics.metric_id
+                    INNER JOIN package_metrics ON metric.id = package_metrics.id
```

Comment:
Why was this changed? `metric_id` is the foreign key from the `metric` table in the `package_metrics` table.

Reply:
Because `metric_id` was all NA and it wasn't working for me.